.htaccess

```diff
@@ -20,10 +20,10 @@ AddType application/json apple-app-site-association
 RewriteRule "^.well-known/apple-app-site-association$" "/.well-known/apple-app-site-association.json" [L]
 
 # macosx and macos to mac (ignore case)
-RedirectMatch 301 "^(?i)/(macosx|macos)\b(.*)$" "/mac$2"
+RewriteRule "^(macosx|macos)\b(.*)$" "/mac$2" [NC,R=301,END,QSA]
```

Comment:
For case-insensitive matches, RedirectMatch uses `(?i) and RewriteRule uses `NC`

Reply:
.htaccess really is an appalling file format, isn't it? So many edge cases and fiddly bits. So much legacy. Seems like a lot of the functionality "just happened" rather than being properly designed.